rasa/server.py

```diff
@@ -852,6 +853,93 @@ async def retrieve_story(request: Request, conversation_id: Text) -> HTTPRespons
                 f"An unexpected error occurred. Error: {e}",
             )
 
+    @app.get("/billable_conversations")
+    @run_in_thread
+    async def billable_conversations(request: Request) -> HTTPResponse:
```

Comment:
An alternative could be to do this live: if you send the data to smth like `reporting.rasa.com` you could do that for every conversation, right when the condition is met. e.g. when a tracker is persisted with `save` (or somewhere in the processor) you could trigger the evaluation if a new session is started and if so, trigger a post to `reporting.rasa.com`, where payload is probably minimal like `{"license_hash": ...}` + maybe some identifier to reason about bugs.